terraform/modules/emblem-app/main.tf

```diff
@@ -57,3 +57,16 @@ resource "google_storage_bucket_iam_member" "sessions-iam" {
   member = "serviceAccount:${google_service_account.website_manager.email}"
   count  = var.deploy_session_bucket ? 1 : 0
 }
+
+## Ops Cloud Build service account permission to update Firestore
+resource "google_project_service_identity" "ops_cloudbuild" {
```

Comment:
Since we're planning to split up into more Cloud Build service accounts in the future, it would be helpful to have a way to reference from IAM binding to Cloud Build trigger or the reverse so we can tease apart permission assignments later on.

Reply:
Agreed, will probably come as a refactor associated with #254